tests/common.bash

```diff
@@ -186,31 +186,27 @@ function clean_env_ctr()
 	if (( count_tasks > 0 )); then
 		die "Can't remove running containers."
 	fi
-
-	kill_kata_components
```

Comment:
I'd say you could still leave it here.

Reply:
The problem is that the purpose of `clean_env_ctr` is only to kill running containers, that's why we moved `kill_kata_components` to the stage where we need to stop kata running processes.